docs/advanced-scheduled-price-changes/README.md

```diff
@@ -33,15 +32,56 @@ Run task manually, and enter a keyword of __schedule__, __list__, __email__, or
 mechanic/user/text
 user/price_changes/start
 user/price_changes/end
+user/price_changes/reset
 ```
 
 [Learn about event subscriptions in Mechanic](https://docs.usemechanic.com/article/408-subscriptions)
 
 ## Documentation
 
-Run task manually, and enter a keyword of __schedule__, __list__, __email__, or __cancel__. The cancel keyword must be followed by the *price change event ID*, which will be generated when an event is successfully scheduled.
-
-## TODO: Docs
+This advanced task allows you to schedule price change events for your store, useful for seasonal and flash sales. To use this task, configure a price change event using the task options, save the task and run it manually, and then enter a keyword of **schedule** in the dialog that appears. An email summary with the price change event ID will be sent upon successful scheduling.
```

Comment:
Can you add this sentence:

🚨  This is an advanced task, intended for technical users. For tasks like these, the Mechanic team only offers support with platform-level issues. For help with task-level issues (e.g. debugging, configuration, customization, monitoring, etc), see [Hire a Mechanic developer](https://learn.mechanic.dev/hire-a-developer).